lib/features/user_action/commentaire/list/action_commentaire_list_reducer.dart

```diff
@@ -1,9 +1,24 @@
+import 'package:pass_emploi_app/features/user_action/commentaire/create/action_commentaire_create_actions.dart';
 import 'package:pass_emploi_app/features/user_action/commentaire/list/action_commentaire_list_actions.dart';
 import 'package:pass_emploi_app/features/user_action/commentaire/list/action_commentaire_list_state.dart';
+import 'package:pass_emploi_app/models/commentaire.dart';
 
 ActionCommentaireListState actionCommentaireListReducer(ActionCommentaireListState current, dynamic action) {
   if (action is ActionCommentaireListLoadingAction) return ActionCommentaireListLoadingState();
   if (action is ActionCommentaireListFailureAction) return ActionCommentaireListFailureState();
   if (action is ActionCommentaireListSuccessAction) return ActionCommentaireListSuccessState(action.comments);
+  if (action is ActionCommentaireCreateSuccessAction && current is ActionCommentaireListSuccessState) {
```

Comment:
pourquoi on a besoin de && current is ActionCommentaireListSuccessState ? je ne vois pas.

Reply:
pour ajouter un comment dans list existant à la mano sans appeler le back